pkg/fanal/analyzer/language/nodejs/npm/testdata/wrong.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
If we just want to test a case where JSON cannot be parsed, can we make this JSON small? For example, `{broken}` is an invalid JSON.

Reply:
Done